app/config/sample-local.json

```diff
@@ -0,0 +1,26 @@
+{
+  "db": {
+    "username": "app",
+    "password": "password123",
+    "port": "25432"
+  },
+  "frontend": {
+    "keycloak": {
+      "clientId": "comfort-frontend-local"
```

Comment:
clientId should be comfort-frontend.
Also need to add in serverUrl.
`  "frontend": {
    "keycloak": {
      "clientId": "comfort-frontend",
      "serverUrl": "http://localhost:28080/auth"
    }
  },
`

Reply:
Thanks and done!  ...Now the app is behaving differently.  Too bad we can't put PRs back into draft mode.